code/modules/economy/cash.dm

```diff
@@ -17,50 +17,34 @@
 	var/worth = 0
 
 /obj/item/weapon/spacecash/c1
-	name = "1 credip chip"
```

Comment:
I just realized how fucking pointless this entire file is, it's not actually compiled